spec/book_spec.rb

```diff
@@ -0,0 +1,81 @@
+require 'rails_helper'
+
+describe Book, type: :model do
+  #  before(:all) do
+  #    @book = create(:book)
+  #  end
+
+  it do
+    should validate_presence_of(:title)
```

Comment:
It's a good way to do test but here you have 2 issues:
1. prefer not using should
2. this i duplicate test.

So here we will keep basic tests :)